cmd/tracee-rules/main.go

```diff
@@ -54,10 +54,15 @@ func main() {
 				return fmt.Errorf("invalid target specified: %s", strings.ToLower(c.String("rego-runtime-target")))
 			}
 
+			var rulesDir []string
+			if c.String("rules-dir") != "" {
+				rulesDir = []string{c.String("rules-dir")}
```

Comment:
Shouldn't this be GetStringArray?

Reply:
We are not changing `tracee-rules`, only `tracee`.